rhombus/draw/private/font.rhm

```diff
@@ -110,6 +110,6 @@ class Font(private _handle):
       #false
 
   fun norm_hash(map :: Map):
-    racket.hash(& (for values(list = []):
+    racket.hash(& (for values(list = PairList[]):
                      each values(k, v): map
-                     [k, v, &list]))
+                     PairList[k, v, &list]))
```

Comment:
Can you add a space after this and other occurrences of `&`, just for a more consistent style?

Reply:
HF